docs/shared/packages/jest/jest-plugin.md

```diff
@@ -161,6 +161,16 @@ To run Jest tests via nx use
 nx test frontend
 ```
 
+### Testing a Single File
```

Comment:
I'd rephrase it a bit because the regex can match multiple files. So, the section is really showing how to test specific files.
```suggestion
### Testing Specific Files
```